qa/rpc-tests/sc_split.py

```diff
@@ -197,60 +178,66 @@ def run_test(self):
 
         # ---------------------------------------------------------------------------------------
         # Nodes 2 start to work on malicious chain
-        print("------------------------------")
-        self.mark_logs("\nNode 2 generates 4 malicious blocks, its chain will have a greater length than honest...")
+        mark_logs("\nNode 2 generates 4 malicious blocks, its chain will have a greater length than honest...",self.nodes,DEBUG_MODE)
 
         blocks.extend(self.nodes[2].generate(4))
         self.sync_all()
 
-        self.mark_logs("\nJoining network")
+        mark_logs("\nJoining network",self.nodes,DEBUG_MODE)
         self.join_network()
-        self.mark_logs("Network joined")
+        mark_logs("Network joined",self.nodes,DEBUG_MODE)
 
-        print ("\nChecking that sc info on Node1 are not available anymore since tx has been reverted...")
+        mark_logs ("\nChecking that sc info on Node1 are not available anymore since tx has been reverted...",self.nodes,DEBUG_MODE)
         try:
-            print (self.nodes[1].getscinfo(scid))
+            mark_logs (self.nodes[1].getscinfo(scid),self.nodes,DEBUG_MODE)
         except JSONRPCException, e:
             errorString = e.error['message']
-            print (errorString)
+            mark_logs (errorString,self.nodes,DEBUG_MODE)
         assert_equal("scid not yet created" in errorString, True)
 
         # Check the mempools of every nodes
-        print ("\nChecking mempools...")
+        mark_logs ("\nChecking mempools...",self.nodes,DEBUG_MODE)
         for i in range(0, NUMB_OF_NODES):
             txmem = self.nodes[i].getrawmempool()
             if i == 2:
                 assert_equal(len(txmem), 0)
             else:
                 assert_equal(sorted(txes), sorted(txmem))
 
+        # Node 1 try to reuse inputs of FT
+        mark_logs("\nNode 1 try to reuse inputs of FT...",self.nodes,DEBUG_MODE)
+
+        try:
+            self.nodes[1].sendtoaddress("zthXuPst7DVeePf2ZQvodgyMfQCrYf9oVx4", 3.5)
+        except JSONRPCException, e:
+            errorString = e.error['message']
+            mark_logs (errorString,self.nodes,DEBUG_MODE)
+        assert_equal("Insufficient funds" in errorString, True)
```

Comment:
With this lines you want to prove that coins used for the forward transfer are not available any more even if the forward transfer has not been confirmed.
Instead of actually trying to spend those UTXOs I would check the balance in node and verify that is is less than 3.5